tests/test_simplelog.py

```diff
@@ -0,0 +1,192 @@
+import sys
+
+import re
+import click
+import logging
+
+log_stream = None
+
+
+def tool_resetlog():
```

Comment:
Since the logger is supposed to print to stdout/stderr, you can use [pytest's features](http://pytest.org/latest/capture.html) for this.

Also, please don't use any global state.